Documentation/compression.md

```diff
@@ -0,0 +1,31 @@
+# Compression
+
+## Client side
+The preferred method for configuring message compression on a client is to pass `options` when the client object is instantiated.
+
+The two options need to be passed:
+
+**grpc.default_compression_algorithm** (integer)
+
+The option keeps the value of a compression algorithm.
+
+Possible values for this option are:
+- `0` - No compression
+- `1` - Compress with DEFLATE algorithm
+- `2` - Compress with GZIP algorithm
+- `3` - Stream compression with GZIP algorithm
+
+**grpc.default_compression_level** (integer)
+
+The option keeps the value for the level of compression.
```

Comment:
Similarly, this should also describe the functionality of this option. I think this one controls the set of compression algorithms the channel will accept when reading messages, but I'm less sure of this one.